x-pack/packages/kbn-ai-assistant/src/chat/welcome_message.tsx

```diff
@@ -39,22 +39,15 @@ export function WelcomeMessage({
 }) {
   const breakpoint = useCurrentEuiBreakpoint();
 
-  const {
-    application: { navigateToApp, capabilities },
-    plugins: {
-      start: {
-        triggersActionsUi: { getAddConnectorFlyout: ConnectorFlyout },
-      },
-    },
-  } = useKibana().services;
+  const { application, triggersActionsUi } = useKibana().services;
```

Comment:
I am a little concerned about this pattern where there are assumptions about what is actually in `KibanaContext` (because there are no guarantees), but I also don't have a better solution.

Reply:
Yeah it's ugly but temporary. Next steps (after I get Search AI Assistant working) will be to rewrite the entry points for these components to provide the necessary functions through component properties instead of assuming context. But I wanted to keep that for a follow-up PR given how big this has already gotten.

Reply:
Ideally we keep using context (unless you want dozens of layers of prop-drilling), but maybe one specific to the AI Assistant that can be typed-checked